lib/ui/survey_question/answer/answer_dropdown.dart

```diff
@@ -13,7 +16,20 @@ class AnswerDropdown extends StatelessWidget {
   });
 
   @override
-  Widget build(BuildContext context) {
+  Widget build(BuildContext context, WidgetRef ref) {
+    var selectedAnswerIndex = 0;
+
+    ref.listen(surveyNextQuestionsProvider, (_, displayType) {
+      if (displayType.value == DisplayType.dropdown) {
```

Comment:
My bad 🙈 @hoangnguyen92dn, I thought it would trigger all widgets' answers when clicking on the next button, but after debugging, I realized it only triggers the current widget that was entered. I will remove it.🚀  Good catch 👍